### PR TITLE
task-NXDOC-2581-configuration-management-endpoint-documentation

### PR DESCRIPTION
**Pull request by NourNuxeo**

### CODE REVIEW

**Comment by NourNuxeo** [comment]
Going to draft:
Forgot "Add mention on this table at the bottom about migration endpoint."

File: src/rest-api/endpoints/management-endpoint/configuration-endpoint.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+If successful, returns a JSON representation of all configured properties and all runtime properties with sensitive informations redacted.
+
+### Status Codes
+
+- 200 _OK_ - Success.
+
+### Sample
+
+```curl
+curl -u <user>:<password> <nuxeoHost>:8080/nuxeo/api/v1/management/configuration
```
Comment (bdelbosc) [inline src/rest-api/endpoints/management-endpoint/configuration-endpoint.md]: Use the same pattern than other endpoint: 
`curl -u Administrator:Administrator http://localhost:8080/...`
A copy and paste should work on a local dev server

File: src/rest-api/endpoints/management-endpoint/configuration-endpoint.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Get a Nuxeo instance's configuration
+
+```
+GET /management/configuration
+```
+
+### Response
+
+If successful, returns a JSON representation of all configured properties and all runtime properties with sensitive informations redacted.
```
Comment (bdelbosc) [inline src/rest-api/endpoints/management-endpoint/configuration-endpoint.md]: Expect to have questions on what is configured vs runtime props, and why runtime props duplicates configured props.

Comment (NourNuxeo) [reply src/rest-api/endpoints/management-endpoint/configuration-endpoint.md]: Do you mean I should explain it here ?

Comment (NourNuxeo) [reply src/rest-api/endpoints/management-endpoint/configuration-endpoint.md]: I have merged the explenation from the ticket in the sentence.

**Review by bdelbosc — APPROVED** [review]

**Comment by bdelbosc** [comment]
Wunderbar!

**Pull request merged**